Strange linker error on macOS

Trying to build with the GUI or without on macOS with support for the ice40 (don't think GUI or target is related there).

Have an issue, at the end get this:
```
[ 92%] Building CXX object CMakeFiles/nextpnr-ice40.dir/ice40/bitstream.cc.o
[ 93%] Building CXX object CMakeFiles/nextpnr-ice40.dir/ice40/cells.cc.o
[ 94%] Building CXX object CMakeFiles/nextpnr-ice40.dir/ice40/chains.cc.o
[ 95%] Building CXX object CMakeFiles/nextpnr-ice40.dir/ice40/delay.cc.o
[ 96%] Building CXX object CMakeFiles/nextpnr-ice40.dir/ice40/gfx.cc.o
[ 97%] Building CXX object CMakeFiles/nextpnr-ice40.dir/ice40/main.cc.o
[ 98%] Building CXX object CMakeFiles/nextpnr-ice40.dir/ice40/pack.cc.o
4 warnings generated.
[ 99%] Building CXX object CMakeFiles/nextpnr-ice40.dir/ice40/pcf.cc.o
[100%] Linking CXX executable nextpnr-ice40
ld: library not found for -ltbb
clang: error: linker command failed with exit code 1 (use -v to see invocation)
make[2]: *** [nextpnr-ice40] Error 1
make[1]: *** [CMakeFiles/nextpnr-ice40.dir/all] Error 2
make: *** [all] Error 2
```

`libtbb` is findable with a normal `pkg-config` invocation and the files seem to be there, so don't really know what's going on here.  

Note that this is an x86 mac running Big Sur. Not sure the OS version is relevant but it's not an M1 mac.

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -257,7 +257,7 @@ if(PROFILER)
     list(APPEND EXTRA_LIB_DEPS profiler)
 endif()
 if(TBB_FOUND)
-    list(APPEND EXTRA_LIB_DEPS tbb)
+    list(APPEND EXTRA_LIB_DEPS TBB::tbb)
 endif()
 
 foreach (family ${ARCH})

